### ISSUE TITLE
Automatically build new images for the harvesters when changes are pushed to the repo

### ISSUE THREAD

**Issue by iprunache** [NRGI/resourcedata.org#195]
## Why
Allow developers to deploy changes to harvesters in production.

## What
- [x] the eiti harvester is rebuild when changes are pushed to the repo and the image is published to Docker Hub
- [x] the rgi harvester is rebuild when changes are pushed to the repo and the image is published to Docker Hub

## Notes
We don't need to trigger a deploy as the EITI harvester is scheduled to run daily. The scheduler pulls the latest EITI harvester image from Docker Hub.

The RGI harvester is only run manually but it's handy to have the image built and published.

Basically we need a `.shippable.yml` file that builds both harvester images and pushes them to Docker Hub.

**Comment by adinuca** [comment]
There is only a master branch so this will be the only one that will trigger the images to build.

**Comment by adinuca** [comment]
Both images are built when a commit is made to the repository. 
The build job can be found here : https://app.shippable.com/github/NRGI/resourcedata.org/dashboard
The images are sent to docker hub : https://hub.docker.com/r/nrgi/resourcedata.org-harvesters/tags/. The new images have the commit hash attached instead of just a number.

**Comment by iprunache** [comment]
Let's create the `staging` branch and test that its deployment works too.
Until now I was creating staging containers manually but we need a way to
allow developers to create them too.


[image: Vitamin Software Logo]

*Ionut Prunache*
ionut@vitaminsoftware.com
Software Engineer
vitaminsoftware.com <http://www.vitaminsoftware.com/>

**Comment by adinuca** [comment]
Created staging branch which triggered the shippable job and new containers for staging harvesters were saved in docker hub :  https://hub.docker.com/r/nrgi/resourcedata.org-harvesters/tags/

**Comment by adinuca** [comment]
Matt was informed of the changes made on this repository.

The eiti harvester job ran this morning.
